plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-name/block.json

```
@@ -54,5 +54,6 @@
 				"padding": true
 			}
 		}
-	}
+	},
+	"usesContext": [ "attributeId", "attributeName", "attributeTerms" ]
```

I think it would be good to prefix the context with the plugin name:

```suggestion
	"usesContext": [ "woocommerce/attributeId", "woocommerce/attributeName", "woocommerce/attributeTerms" ]
```